castervoice/rules/ccr/rust_rules/rust2.py

```diff
@@ -13,12 +13,16 @@ class RustNon(MappingRule):
             R(Text("panic!()") + Key("left")),
         "macro assertion":
             R(Text("assert_eq!()") + Key("left")),
+        "macro debug":
+            R(Text("dbg!(&)") + Key("left")),
```

Comment:
Debug is a relatively new macro in rust, and was probably not available when the grammar was originally defined. Most often when debugging, it is useful to pass the value by reference. That's why it contains an &